### PR TITLE
Update Ewald error formula

### PR DESCRIPTION
**Pull request by GardevoirX**
Fixes the error formula of Ewald used in tuning. The fixed formula is tested with systems of crystal cells repeated 8 times on each dimension.

No difference in the actual accuracy is observed.
acc

The Ewald calculation time is a bit longer with the fixed formula.
ewald_calc

# Contributor (creator of pull-request) checklist

 - [ ] Tests updated (for new features and bugfixes)?
 - [ ] Documentation updated (for new features)?
 - [x] Issue referenced (for PRs that solve an issue)?

# Reviewer checklist

 - [ ] CHANGELOG updated with public API or any other important changes?

----
📚 Documentation preview 📚: https://torch-pme--198.org.readthedocs.build/en/198/

### CODE REVIEW

File: src/torchpme/tuning/ewald.py [hunk 1]
```diff
@@ -174,8 +174,8 @@ def err_kspace(
         return (
             self.prefac**0.5
             / smearing
-            / torch.sqrt((2 * torch.pi) ** 2 * self.volume / (lr_wavelength) ** 0.5)
-            * torch.exp(-((2 * torch.pi) ** 2) * smearing**2 / (lr_wavelength))
+            / torch.sqrt(2 * torch.pi**2 * self.volume / lr_wavelength)
+            * torch.exp(-2 * torch.pi**2 * smearing**2 / lr_wavelength**2)
```
Comment (PicoCentauri) [inline src/torchpme/tuning/ewald.py:178]: Are you sure this change is correct? I think units still don't match because `smearing` and the `lr_wavelength` should cancel. I just re-derived the equation and I got

$$
ΔF_\mathrm{LR} = \frac{Q^2}{\sqrt{N}} \frac{h}{π \sqrt{V} \sigma } \mathrm{exp} \left( \frac{2 π \sigma^2}{h^2}\right)
$$

maybe check it again.

I think we should also give the conversion between our convention and the other usual one somewhere in the docs:

$$
α = \frac{1}{\sqrt{2}Σ}
K = \frac{2π}{h}
$$

where α is the inverse width of Gaussian charge cloud and $K$ is the Fourier space cutoff in inverse units of length.

**Review by PicoCentauri — CHANGES_REQUESTED** [review]
The docs should also be updated with the change.

**Review by PicoCentauri — APPROVED** [review]
Thanks for the fix @GardevoirX!
